controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -174,24 +201,97 @@ func (r *StorageConsumerReconciler) reconcilePhases() (reconcile.Result, error)
 	return reconcile.Result{}, nil
 }
 
+func (r *StorageConsumerReconciler) reconcileCephRadosNamespace(
+	consumerConfigMap *corev1.ConfigMap,
+	radosnamespace string,
+) error {
+	blockPools := &rookCephv1.CephBlockPoolList{}
+	if err := r.List(r.ctx, blockPools, client.InNamespace(r.namespace)); err != nil {
+		return err
+	}
+
+	var combinedError error
+
+	// ensure for this consumer a rados namespace is created in every blockpool
+	for idx := range blockPools.Items {
```

Comment:
Reframe from using the full term Error for errors and declare the var as close as possible to where it is going to be used. 
```suggestion

	// ensure for this consumer a rados namespace is created in every blockpool
	var combinedErr error
	for idx := range blockPools.Items {
```